resources/store/index.js

```diff
@@ -77,22 +77,14 @@ module.exports = new Vuex.Store( {
 
 		user: {
 			isAuthenticated: !!mw.config.get( 'wgUserName' ),
-			isAutoConfirmed: userGroups.indexOf( 'autoconfirmed' ) !== -1
+			isAutoconfirmed: userGroups.indexOf( 'autoconfirmed' ) !== -1
 		}
 	},
 
 	/**
 	 * Getters are like computed properties for Vuex state
 	 */
-	getters: {
-		/**
-		 * @param {Object} state
-		 * @return {bool}
-		 */
-		showTabs: function ( state ) {
-			return state.user.isAuthenticated && state.user.isAutoConfirmed;
-		}
-	},
+	getters: {},
```

Comment:
Leaving a placeholder here is good because we might end up adding something else here as we continue building out the front-end. But you can go ahead and remove the `mapGetters` import in App.vue for now.